mock_server.py

```
@@ -0,0 +1,407 @@
+if __name__ == "__main__":
+    sys.stdout.reconfigure(line_buffering=True)  # 파일로 리다이렉트해도 요청 로그가 즉시 보이게
+    if not MOCK_IMAGES_DIR.is_dir():
+        sys.exit(f"MockImages 디렉토리를 찾을 수 없음: {MOCK_IMAGES_DIR}")
+    print(f"Rephoto 목 서버 시작 — http://127.0.0.1:{PORT}")
+    print(f"이미지 디렉토리: {MOCK_IMAGES_DIR}")
+    if DELAY_MS:
+        print(f"응답 지연: {DELAY_MS}ms")
+    print("디버그: POST /debug/expire (401→리프레시), POST /debug/expire-all (forceLogout)")
+    print("종료: Ctrl+C")
+    try:
+        ThreadingHTTPServer(("0.0.0.0", PORT), Handler).serve_forever()
```

Ruff (0.16.2)

[error] Possible binding to all interfaces

(S104)